salt/tops/varstack.py

```diff
@@ -45,27 +45,20 @@
 
 from __future__ import absolute_import, print_function, unicode_literals
 
-# Import python libs
-import logging
-
 try:
     HAS_VARSTACK = False
     import varstack
     HAS_VARSTACK = True
 except ImportError:
     pass
 
```

Comment:
If you already doing this, then also please remove these `HAS_FOO` vars as well. As simple as that:

```python
try:
    import varstack
except ImportError:
    varstack = None

def __virtual__():
    return bool(varstack), 'The varstack module could not be loaded: varstack dependency is missing.'
```

Note, I've updated the error message. Feel free to just copypaste all that. :wink: